dependencies/Indy10/System/IdStack.pas

```diff
@@ -613,11 +648,15 @@ constructor TIdStack.Create;
 begin
   // Here for .net
   inherited Create;
+  FHostNames := TStringList.create;

```

Comment:
FHostNames is declared as TObjectList, but is being assigned a TStringList instead.